### ISSUE TITLE
Custom loading indicators don't receive warning messages just disappear

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#3177]
### Describe the bug?

When a user has a custom loading indicator It will not display a deny message it just disappears without describing why the user wasn't able to enter the session

### To Reproduce

1. Save the loading indicator from the Resonite essentials
2.  set it as your default
3. Try joining a session that has too many users or Use a custom join verifier

### Expected behavior

Custom loading indicators should receive error messages

### Screenshots

https://github.com/user-attachments/assets/1bf163e9-bf31-403c-8e31-502dae7be203

### Resonite Version Number

Beta 2024.11.12.1329

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[EPICEASTON197 - 2024.11.12.1329 - 2024-11-13 15_26_32.log](https://github.com/user-attachments/files/17738865/EPICEASTON197.-.2024.11.12.1329.-.2024-11-13.15_26_32.log)

### Reporters

@epicEaston197

**Comment by Frooxius** [comment]
I don't quite understand this issue. There's nothing "special" about custom indicators - they all use the exact same interface.

If it doesn't show an error message, it's probably just not handling that state properly - what leads you to believe that it doesn't actually receive the error message?

- Does this not happen if you use the default one?
- Does this happen with every single custom indicator?

**Comment by epicEaston197** [comment]
>Does this not happen if you use the default one? 

Yes and no if you don't have a loading indicator favorited it will properly show error messages like Access denied but if you save the default indicator and set it as your custom indicator you will not receive such messages

>Does this happen with every single custom indicator?

I'm not entirely sure I'm pretty sure the answer is yes since favoriting the default indicator from the essentials folder results in not receiving error messages

**Comment by JackTheFoxOtter** [comment]
Sounds more like the copy in Resonite Essentials is just broken.

**Comment by epicEaston197** [comment]
> Sounds more like the copy in Resonite Essentials is just broken.

I don't think any custom loading indicator would just disappear like that

**Comment by Frooxius** [comment]
The mechanism for the default and custom is exactly the same, so I'm a bit lost on why would custom ones have the bug. Technically all of them are "custom", even the default one.

It would help to find definitive answers for my questions for this.

**Comment by epicEaston197** [comment]
I don't have any answers because all that I'm doing is saving the existing loading indicator and favoriting it. 
Have you had any luck recreating the issue with my replication steps?.

**Comment by Frooxius** [comment]
I didn't have time to recreate issue.

Generally I will not start recreating issues until I got all the information I need.

Which exact indicator from the folder are you favoriting?

**Comment by epicEaston197** [comment]
> Which exact indicator from the folder are you favoriting?

The one from Resonite Essentials>Default UI

**Comment by paradoxical-autumn** [comment]
I've been using custom indicators for months and couldn't replicate this.
Here's the one I used: [WorldLoadingIndicator.zip](https://github.com/user-attachments/files/17795467/WorldLoadingIndicator.zip)

I then tested the one in Resonite essentials and it was failing to get the error state, so this is an issue with the one stored in there. _(It might seem as if "all" indicators are broken as many people just edit the default one they find in the essentials folder)_

**Comment by epicEaston197** [comment]
> I then tested the one in Resonite essentials and it was failing to get the error state, so this is an issue with the one stored in there. _(It might seem as if "all" indicators are broken as many people just edit the default one they find in the essentials folder)_

Ah OK this means that this needs to be marked as a content Issue and have the appropriate people assigned

**Comment by epicEaston197** [comment]
I've edited the issue to better reflect the problem As a content issue

**Comment by shiftyscales** [comment]
Seeking input from @AegisTheWolf @RueShejn or @RyuviTheViali - it sounds like the copy of this interface in the essentials folder is out of step with the live copies.
